ftw/theming/resources/scss/globals/variables.scss

```diff
@@ -137,61 +137,35 @@ $spinner-url: "" !default;
   line-height-base,
   spinner-url);
 
-/* Viewports */
-
-// Taken from http://en.wikipedia.org/wiki/Display_resolution
-$desktop-WQHD-width:   2560px !default;
-$desktop-FHD-width:    1920px !default;
-$desktop-HDPLUS-width: 1600px !default;
-$desktop-HD-width:     1360px !default;
-$desktop-WXGA-width:   1280px !default;
-$desktop-XGA-width:    1024px !default;
-
-// Taken from http://www.ciop.com/common-resolutions-for-mobile-phones-and-tablets
-$mobile-large: 1280px !default;
-$mobile-medium: 1024px !default;
-$mobile-small: 800px !default;
-$mobile-smaller: 600px !default;
-$mobile-tiny: 480px !default;
```

Comment:
What's the reason for moving that down?

Reply:
I thought it would be a good idea to have the relevant variables on the top and the deprecated ones on the bottom.